Machine_Learning/Unsupervised_Machine_Learning/ML_3_5_Hierarchical_Clustering_(V).md

```diff
@@ -0,0 +1,11 @@
+
+
+
+## **What is Hierarchical Clustering?**
+
+- Hierarchical clustering, also known as hierarchical cluster analysis, is an algorithm that groups similar objects into groups called clusters.
+- The output is a set of clusters, where each cluster is distinct from each other cluster, and the objects within each cluster are broadly similar to each other.
```

Comment:
remove "from each other cluster" after distinct.

Reply:
@Isha307 , I have made the changes Mam, can you please merge the PR.